### DISCUSSION TITLE
Found a working workaround! [SCAM}

### DISCUSSION THREAD

**Discussion by xHydra1337** [3kh0/ext-remover#139] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Ive really been fucking around with the code for exploit.js, and something didn't really seem right to me, I have the updated chrome browser on my personal computer and it was working just fine, so I knew the script was fine and compatible. I tried so many different things and finally I found one that worked. 

Steps:
1. Open [Google.com/chromes](https://google.com/chromes) (this is an unblocked 404 page)
2. Copy everything from exploit.js
3. ctrl + Shift + i, go to console and paste it in.
4. Your done! 

If this doesn't work for you then I don't really know how to help you.

Edit: When your first paste in console and click enter, its not going to do anything, paste it again and click enter once more and it will work.

**Comment by The-Man-Is-Me** [comment]
still does not work for me, shows gui but does not disable it
![Screenshot 2022-10-25 3 35 16 PM](https://user-images.githubusercontent.com/78439704/197876582-0e283031-99fd-4efe-b13e-f080359f2738.png)
![Screenshot 2022-10-25 3 35 02 PM](https://user-images.githubusercontent.com/78439704/197876588-046c3ed0-9688-4b02-a3b8-f9e77d7b7bcd.png)

**Reply by Sxri** [reply]
He lied, it won't disable it.

**Reply by ghost** [reply]
https://github.com/yeeteeyt/GoGuardian-bypass

**Comment by Sxri** [comment]
This is obviously not going to work. Either you're lying or joking around, but this literally wouldn't work. it's the exact same steps as the main method but with a different URL. Which fixes nothing. 

I don't know how you managed to get it to work, but that probably means your school system is autistic or something, because that wouldn't work.

(not trying to make this come off as aggressive or rude, its just literal common sense)

p.s. it has nothing to do with the code

**Reply by xHydra1337** [reply]
I am genuinely not lying or joking, it works 100% fine for me
Before disabling:
![Screenshot 2022-10-25 14 40 29](https://user-images.githubusercontent.com/116680143/197887639-58b57fd7-7706-423d-a1ed-2f3cbca9290b.png)


![Screenshot 2022-10-25 14 40 43](https://user-images.githubusercontent.com/116680143/197887699-504048a5-48eb-4048-abf2-49c7414a297c.png)

After disabling:
![Screenshot 2022-10-25 14 40 58](https://user-images.githubusercontent.com/116680143/197887761-8c07c0da-c6a4-43b4-aff8-ed6fb916d4ff.png)

![Screenshot 2022-10-25 14 43 05](https://user-images.githubusercontent.com/116680143/197887845-eb4589fb-a4c0-434a-bff3-9328eb1f8f1c.png)


Try this instead: 
https://notepad.link/share/D1QuG0eVZ0kuVbBsyRum

**Reply by ghost** [reply]
u didn't update CHROMEOS dum dum

**Reply by xHydra1337** [reply]
well, school chromebook is set to auto update, or check for updates every time the chromebook is turned back on, so your right, nothing to do with the code, but the reason why the steps in the original README weren't working was because MOST of the policies of these chromebooks are set to not allow page redirects on blocked pages, but if we are able to force the page run the script through console, then it will work just fine. I don't know whats going on for you guys, but this works for me as well as everyone of my friends that I have shown it to.

**Reply by ghost** [reply]
it won't "force" anything

**Comment by ghost** [comment]
This is a lie. Stop circulating lies or your posts will be deleted

**Reply by xHydra1337** [reply]
I'm sorry, I don't know why you keep saying it's a lie or a joke, it's genuinely neither, I guess it just doesn't work for you, your restrictions may be a little different than mine, but it does work for me, that's not saying it works for everybody.

**Reply by ghost** [reply]
it doesn't work because google itself patched the code. you didn't use/update CHROMEOS, and you have dev mode

**Reply by xHydra1337** [reply]
I'm running phaser/octopus board on firmware 107, and my chromebook is not running in dev mode I can assure you, and yes I've tried booting my chromebook in dev mode multiple times with no success. (If it's dev mode for extensions, then yes, I have that ticked)

**Reply by ghost** [reply]
well that's not a working fix

**Comment by Meeep1** [comment]
This works completely fine with my Chromebook, all of the extensions are turned off so It depends on your Chromebook I guess
